.backportrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "prTitle": "{{sourcePullRequest.title}} [backport: {{targetBranch}}]",
```

Comment:
I found that the backport PR title also needs to follow [the Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) so Release Please can create release PRs. I changed the backport PR title to use the original one.